services/authenticator.js

```diff
@@ -5,11 +5,14 @@ const chalk = require('chalk');
 const inquirer = require('inquirer');
 const api = require('./api');
 const { parseJwt } = require('../utils/authenticator-helper');
+const { terminate } = require('../utils/terminator');
 const logger = require('./logger');
 
 const FORMAT_PASSWORD = /^(?=\S*?[A-Z])(?=\S*?[a-z])((?=\S*?[0-9]))\S{8,}$/;
 
 function Authenticator() {
+  const unexpectedError = 'An unexpected error occured. Please create a Github issue with following error:';
```

Comment:
An unexpected error occur**R**ed. Please **reach out for help in our Slack community or** create a Github issue with following error: